ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge2A.cfg

```diff
@@ -364,29 +364,29 @@ route-map RM-BGP-UNDERLAY-PEERS-IN permit 10
    description Allow WAN HA peer interface prefixes
    match ip address prefix-list PL-WAN-HA-PEER-PREFIXES
 !
-route-map RM-BGP-UNDERLAY-PEERS-IN permit 20
-   description Allow prefixes originated from the HA peer
-   match extcommunity ECL-EVPN-SOO
-   set as-path match all replacement auto auto
-!
-route-map RM-BGP-UNDERLAY-PEERS-IN permit 30
-   description Use WAN routes from HA peer as backup
+route-map RM-BGP-UNDERLAY-PEERS-IN deny 20
+   description Deny other routes from the HA peer
    match as-path ASPATH-WAN
-   set community no-advertise
 !
 route-map RM-BGP-UNDERLAY-PEERS-IN permit 40
    description Mark prefixes originated from the LAN
    set extcommunity soo 192.168.42.2:423 additive
 !
 route-map RM-BGP-UNDERLAY-PEERS-OUT permit 10
-   description Advertise local routes towards LAN
-   match extcommunity ECL-EVPN-SOO
+   description Make tagged routes received from WAN HA peer less preferred for LAN router
+   match route-type internal
+   match tag 50
+   set metric 50
 !
 route-map RM-BGP-UNDERLAY-PEERS-OUT permit 20
+   description Advertise local routes towards LAN
+   match route-type local
```

Comment:
we are missing routes learned from an l3_interface or other edge kind of interface with dynamic routing, like a firewall.
Maybe we should set a tag everywhere instead of these route-type matches. It will also be useful for the ospf lan case.

Reply:
l3_interface learned route can configure an extra route-map entry right? This may not even be a use case today.
For OSPF we can use the same route-map I think

Reply:
Fixed in [9d5e835](https://github.com/aristanetworks/avd/pull/3720/commits/9d5e835224aaac3964ed4e41e25be53bbc74f003)